src/webui/www/private/views/rssDownloader.html

```diff
@@ -478,6 +503,13 @@
             });
         };
 
+        const removeSelectedRule = () => {
+            if (rssDownloaderRulesTable.selectedRows.length > 0) {
```

Comment:
I would prefer using this style:
```js
if (rssDownloaderRulesTable.selectedRows.length <= 0)  // is there an `empty()` method for selectedRows?
    return;
// the rest
```

Reply:
There is no empty() method, but this could be used:
```js
if (!rssDownloaderRulesTable.selectedRows.length)
    return;
```

I think this describes more clearly what the line does.

```js
if (rssDownloaderRulesTable.selectedRows.length === 0)
    return;
```

Reply:
> ```js
> if (rssDownloaderRulesTable.selectedRows.length === 0)
>     return;
> ```

👍